wiki/Tournaments/otst!/2019/en.md

```diff
@@ -8,7 +8,7 @@ tags:
 
 ![otst! 2019 Logo](img/logo.png)
 
-The **osu! TV Size Tournament 2019** (***otst! 2019***) is a double elimination four-on-four osu! tournament hosted by ![][flag_TH] [Trigonoculus](/users/7627013) and ![][flag_TH] [Seaweed](/users/5151214). osu! TV Size Tournament features variety of TV sized (usually 90 seconds) maps in its mappools. It is the third installment of the osu! TV Size Tournament.
+The **osu! TV Size Tournament 2019** (***otst! 2019***) is a double-elimination four-on-four osu! tournament hosted by ![][flag_TH] [Trigonoculus](/users/7627013) and ![][flag_TH] [Seaweed](/users/5151214). osu! TV Size Tournament features variety of TV sized (usually 90 seconds) maps in its mappools. It is the third installment of the osu! TV Size Tournament.
```

Comment:
seems like you missed part of my earlier review on this line? or you just didn't respond idk